### ISSUE TITLE
"Media could not be loaded" when loading a playlist using mobile Chrome

### ISSUE THREAD

**Issue by adnankorkmaz** [videojs/videojs-contrib-hls#1084]
"The media could not be loaded, either because the server or network failed or because the format is not supported."

I keep getting this error with the following combinations:
- Mobile android+chrome
- Trying to load a playlist, with .m3u8 extension

First I created a test page. However upon further testing, I could reproduce the error on https://videojs.github.io/videojs-contrib-hls/ demo page.

Steps to reproduce (on desktop Chrome browser):
1. Use Chrome (mine is 57.0.2987.133, but I suspect other versions also encounter the same issue)
2. Hit F12 to activate Chrome DevTools
3. Click "Toggle Device toolbar" or ctrl + shift + m to emulate mobile user agent.
4. From the mobile device list, choose an android device, like "Galaxy S5"
5. Navigate to https://videojs.github.io/videojs-contrib-hls/

![videojs_error](https://cloud.githubusercontent.com/assets/1922822/24952980/4a15762c-1f82-11e7-924d-984531f53005.jpg)

Compared the request and response headers in desktop / mobile requests.
The first difference I noticed was the response status'es (200 vs 206).
And when filtered the network tab of the DevTools with "m3u8", the mobile request's response seems empty with http status code 206.

**Comment by gesinger** [comment]
Hey @adnankorkmaz , thanks for reporting the issue, and for providing reproduction steps. For Chrome's device emulation though, it doesn't provide exact device emulation, only pieces of the device (e.g., it helps for responsive web debugging and handling device input -- https://developers.google.com/web/tools/chrome-devtools/device-mode/emulate-mobile-viewports).

Android Chrome should be using native playback for HLS, unless override native is used (https://github.com/videojs/videojs-contrib-hls#overridenative). It may help if you could identify the Android device you are specifically testing the sample page on, as it could be a bug with their native playback. Thanks.

**Comment by adnankorkmaz** [comment]
Thanks @gesinger for the response.

I had said in my first comment that I had "_created a test page to reproduce the error_", different than the [demo page](https://videojs.github.io/videojs-contrib-hls/). 

The original error-reproduction page involved Google's [ima3.js](https://imasdk.googleapis.com/js/sdkloader/ima3.js). 
It is a rare combination of videojs + videojs-contrib-hls + mobile chrome + autoplay + m3u8 playlist + preroll ad request.

Upon further investigating and toggling scripts in turn, the real issue turned out to be a concurrency problem where IMA and videojs was trying to initialize the same player; and that unmuted autoPlaying videos on Mobile was [not allowed](https://developers.google.com/web/updates/2016/07/autoplay) without user interaction first. Not quite having solved the real issue, I am pretty sure that it doesn't happen due to video-*.js at the moment.

(Added details for possible future developers facing the same issue, however unlikely)

TLDR, closing the issue.

**Comment by gesinger** [comment]
Thanks for the update @adnankorkmaz !
